releases/v0.5.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.5.0
+Release Date: 2023-03-14
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
+- Improved validation coverage
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+- Intel SGX
+
+The following platforms are untested or partially supported:
+- AMD SEV-ES
+- IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+
+The following platforms are in development:
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * AMD SEV-ES is not tested in the CI.
+  * Image signature validation has not been tested with AMD SEV.
+  * s390x does not support cosign signature validation
+- SELinux is not supported on the host and must be set to permissive if in use.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used for non-tee testing, but is not suitable for
+  production, except with encrypted VM images.
+  * Currently, there are two key broker services (KBS) that can be used:
+    - simple-kbs:  simple key broker service for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which
+    Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
+- The format of encrypted container images is still subject to change
+  * The [oci-crypt](https://github.com/containers/ocicrypt) container image format itself may still change
+  * The tools to generate images are not in their final form
+  * The image format itself is subject to change in upcoming releases
```

Comment:
In v0.5.0, 
- we will have a [unified format of encrypted image for CoCo](https://github.com/confidential-containers/attestation-agent/issues/113)

Reply:
Let's take this item to the next release notes PR. I'm less clear how to handle the text for this one and this PR is starting to get a little messy.